implementations/python/sugar-source-tree/src/sugar_source_tree/bench_backends.py

```
@@ -0,0 +1,176 @@
+    load_start = _load1()
+    t0 = time.perf_counter()
+    constructed = 0
+    nodes = 0
+    for rel, source in sources:
+        try:
+            file = SourceFile(filename=rel, source=source, backend=tree.backend)
+            for _n in file.nodes():
+                nodes += 1
+            constructed += 1
+        except Exception:
+            continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)